Project.toml

```
@@ -1,6 +1,6 @@
 name = "Turing"
 uuid = "fce5fe82-541a-59a6-adf8-730c64b5f9a0"
-version = "0.15.15"
+version = "0.15.16"
```

Since 0.15.16 was just released this morning I'll update it to
```suggestion
version = "0.15.17"
```